### PR TITLE
[DR-1596] Add ready step 1 message

### PR DESCRIPTION
**Pull request by cristiangiagante**
## Background
  
Add a message to show the ready step 1 and pending step 2
  
## Changes done
  
Update styles
Add texts to language file
Update plans view
Update test
Add style to the list number

## Pending to be done
  
N/A
  
## Notes
  
@mvillaseco What do you think about the changes? 
 
## Demo
 
image

image

image

image

### CODE REVIEW

File: src/wwwroot/styles/views/_settings.scss [hunk 1]
```diff
@@ -656,19 +656,29 @@
     ol{
       padding-left: 15px;
       margin-bottom: 0px;
+      color: $suit-accent-color;
       li{
         counter-increment: list;
         position: relative;
         text-align: left;
+        padding-top: 5px;
```
Comment (mvillaseco) [inline src/wwwroot/styles/views/_settings.scss]: is this necessary?

File: src/wwwroot/styles/views/_settings.scss [hunk 2]
```diff
@@ -656,19 +656,29 @@
     ol{
       padding-left: 15px;
       margin-bottom: 0px;
```
Comment (mvillaseco) [inline src/wwwroot/styles/views/_settings.scss]: Wasn't this in the reset?

Comment (cristiangiagante) [reply src/wwwroot/styles/views/_settings.scss]: checking the reset _normalize.scss I found:
```css
  dl,
  menu,
  ol,
  ul {
    *margin: 1em 0;
  }
```

I need to put zero to the 1em.

File: src/wwwroot/styles/views/_settings.scss (hunk 1 above)
Comment (mvillaseco) [reply src/wwwroot/styles/views/_settings.scss]: No you don't, padding i'ts not the right call here, use margin instead -> https://stackoverflow.com/questions/2189452/when-to-use-margin-vs-padding-in-css

File: src/wwwroot/styles/views/_settings.scss [hunk 3]
```diff
@@ -656,19 +656,29 @@
...
         &::before{
-          color: $suit-accent-color;
           content: counter(list) ".";
           left:-30px;
           position: absolute;
           text-align: right;
           width: 26px;
           font-weight: bolder;
         }
+        span{
```
[the remark this answers is not in the document]
Comment (cristiangiagante) [reply src/wwwroot/styles/views/_settings.scss]: Thanks mauro I changed it

**Comment by cristiangiagante** [comment]
The current screenshots are updated with this changes.

**Pull request merged**